src/main/java/Rename.java

```diff
@@ -0,0 +1,6 @@
+public class Rename {
+    public static void main(String[] args){
+        String name = args[0];
+        System.out.println("Hello! I'm " + name);
+    }
+}
```

Comment:
May I ask the what is the purpose of this class? I don't see it being used anywhere